### PR TITLE
docs: Update User Guide for findPerson and findTask

### PR DESCRIPTION
**Pull request by jiakai-17**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [x] Documentation Update

## Description

* Add command format, notes, examples and screenshots
* Replace some admonitions
* Remove placeholder code

## Related Tickets & Documents

-   Related Issue #38 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [ ] Yes
-   [x] No, and this is why: See deployment: https://docs-jiakai-user-guide-v1-1.coordimate.pages.dev/DraftUserGuide
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

File: docs/DraftUserGuide.md [hunk 1]
```diff
@@ -203,23 +212,29 @@ Errors:
...
+This command hides all persons that do not match the search criteria. <br>
+To reset the Persons view, simply run the <code>listPerson</code> command to list all persons.
+
+" %}
 
 Format:
 
 ```
-findPerson KEYWORD [MORE_KEYWORDS]
+findPerson KEYWORD [KEYWORDS]…
```
Comment (HugeNoob) [inline docs/DraftUserGuide.md:229]: I actually think `MORE_KEYWORDS` is better. Makes it clear that this isn't a typo.

File: docs/DraftUserGuide.md [hunk 2]
```diff
@@ -351,37 +368,51 @@ Output:
...
+This command hides all Tasks that do not match the search criteria. <br>
+To reset the Tasks view, simply run the <code>listTasks</code> command to list all Tasks.
+
+" %}
 
 Format:
 
 ```
-findTask KEYWORD [MORE_KEYWORDS]
+findTask t/KEYWORD [KEYWORDS]… [n/KEYWORD [KEYWORDS]…]
```
Comment (HugeNoob) [inline docs/DraftUserGuide.md:385]: As discussed, perhaps we can reduce the work by collapsing this functionality to search through title and docs together.

Remember to change the documentation below to match the changes.

File: docs/images/output/findTask_success.png
Comment (HugeNoob) [inline docs/images/output/findTask_success.png:1]: Would be good if you match the images to the examples you listed.

**Review by HugeNoob — CHANGES_REQUESTED** [review]

**Review by freshcabbage123 — APPROVED** [review]
Looks good, thank you for factoring in the suggested changes!

**Review by m1oojv — APPROVED** [review]
LGTM nice job

**Comment by jiakai-17** [comment]
Superseded by #70

**Pull request closed without merging**